README.md

```diff
@@ -17,8 +17,15 @@ and public blockchain interoperability, Dragonchain shines a new and interesting
 
 </div>
 
+## 🏁 Getting Started
```

Comment:
is this a visual bug or intended? the image just looks like some scribbles to me

Reply:
Its the checker flag emoji